pcsx2/Docs/License.txt

```diff
@@ -4,7 +4,7 @@
 
 
 /*  PCSX2 - PS2 Emulator for PCs
- *  Copyright (C) 2002-2012  PCSX2 Dev Team
+ *  Copyright (C) 2002-2019  PCSX2 Dev Team
```

Comment:
Not necessary to change.

Reply:
Perhaps not necessary - but correct, yes?

Reply:
Maybe not necessary but this is still correct. Sticking with it for now.